src/components/ReportingConfig/ReportingConfigForm.jsx

```diff
@@ -59,6 +60,12 @@ class ReportingConfigForm extends React.Component {
     const invalidFields = requiredFields
       .filter(field => !formData.get(field))
       .reduce((prevFields, currField) => ({ ...prevFields, [currField]: true }), {});
+    if (!Object.prototype.hasOwnProperty.call(invalidFields, 'pgpEncryptionKey')) {
```

Comment:
I think following makes more sense in this case.
```
if(!formData.get("pgpEncryptionKey")) {
   if (!formData.get("encryptedPassword")) {
        invalidFields.encryptedPassword = true;
   }
}

if(!formData.get("encryptedPassword")) {
   if (!formData.get("pgpEncryptionKey")) {
        invalidFields.pgpEncryptionKey = true;
   }
}
```

Reply:
ok. I try to do that and analyze the results.